src/entities/message-template/message-template.ts

```diff
@@ -79,6 +79,38 @@ export interface MessageTemplateRawPayloadLocalizedContentItemAction {
   utm_campaign?: string
 }
 
+export interface MessageTemplateRawPayloadQuickReplies {
+  translate?: boolean
+  translation_prepend?: string | null
+  translation_append?: string | null
+  replies?: MessageTemplateRawPayloadQuickReply[]
+}
+
+export interface MessageTemplateRawPayloadList {
+  body?: string
+  type?: 'list' | string | null
+  button?: {
+    text?: string | null
+    type?: string | null
+    button_type?: string | null
```

Comment:
```suggestion
    button_type?: string | null
    utm_source?: string | null
    utm_medium?: string | null
    utm_campaign? string | null
    utm_id?: string | null
    utm_term?: string | null
    utm_content?: string | null
```

Reply:
this is probably not in the scope of that PR but we could add those here for the Link Manager as well ? wdyt @artdnlv

Reply:
ah that the list type `MessageTemplateRawPayloadList` nvm